docs/field-details.asciidoc

```diff
@@ -7554,6 +7554,202 @@ example: `MITRE ATT&CK`
 
 // ===============================================================
 
+|
+[[field-threat-group-alias]]
+<<field-threat-group-alias, threat.group.alias>>
+
+| beta:[ This field is beta and subject to change. ]
+
+The alias(es) of the group for a set of related intrusion activity that are tracked by a common name in the security community. While not required, you can use a MITRE ATT&CK® group alias(es).
+
+type: keyword
+
+
+Note: this field should contain an array of values.
```

Comment:
Is it accurate to say that a single value would be held in the first position of an array? If so, I think "may".

Reply:
Agreed. May is correct.

I'll make this change.

Reply:
This isn't something that can be changed ad-hoc. It's added to a field description automatically based on this attribute in the field's definition:

```
normalize:
  - array
```

You might have noticed the tests running on CI came back ❌ since CI will check for any uncommitted deltas with generated files.

See my note https://github.com/elastic/ecs/pull/1438#issuecomment-849698493. I'm happy to discuss more how we could better phrase when a field is expected to be an array vs a single value, but since it would be a ECS-wide change for many fields I see it as out-of-scope for this PR.

Reply:
I did notice that.

Do you want me to change it back to `should`?

Reply:
@peasead no need. Please leave as is, thanks to @ebeahan for clarification below

Reply:
I took care of it. Thanks, @peasead @djptek @devonakerr